<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchView</class>
 <widget class="QWidget" name="SearchView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>603</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SearchView</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>36</height>
      </size>
     </property>
     <property name="title">
      <string>过滤条件</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>IP：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ip"/>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>开始时间</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="begin_time"/>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>结束时间</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="end_time"/>
      </item>
      <item>
       <widget class="QPushButton" name="btn_find">
        <property name="text">
         <string>查找</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="log_edit"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
